package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "publish-please",
-  "version": "3.2.0",
+  "version": "3.3.0",
```

Comment:
Considering that it can be a breaking change for some users it's worth bumping a major version.